### ISSUE TITLE
Text invisible in Qt apps on Fedora 38 in Mint-Y Dark

### ISSUE THREAD

**Issue by jrefior** [linuxmint/cinnamon#11817]
### Distribution

Fedora 38 Cinnamon Spin

### Package version

5.8.4

### Graphics hardware in use

nVidia geforce gtx 970

### Frequency

Always

### Bug description

After upgrading Fedora 37 => 38 on Fedora Cinnamon Spin, most text in Qt apps is invisible (please see screenshots of VLC video player app) if my selected theme is Mint-Y and Appearance Dark. If I go to system preferences => Theme => and change appearance to Light or Mixed, then text is visible again. Similarly, if I change Theme to Adwaita and choose Appearance Dark, then text is visible again as well.

At first I mentioned this issue [on VLC forums](https://forum.videolan.org/viewtopic.php?f=13&t=162902&p=538823#p538823), a developer thought it might be a Qt issue so I reported [a bug there](https://bugreports.qt.io/browse/QTBUG-115937), and then tonight I discovered changing theme from Mint-Y to Adwaita could resolve it as a workaround, so I'm adding a bug report here.

![Screenshot from 2023-07-29 12-44-06](https://github.com/linuxmint/cinnamon/assets/1707476/31b5927f-6e7c-4a03-8697-ee087ddf0e3a)
![Screenshot from 2023-07-29 12-45-12](https://github.com/linuxmint/cinnamon/assets/1707476/61d7900c-ad94-4a11-a366-3b1ed6e85ac7)
![Screenshot from 2023-07-29 12-56-33](https://github.com/linuxmint/cinnamon/assets/1707476/63487533-0abf-4445-9325-dbf47ab17ee0)
![Screenshot from 2023-08-08 22-11-37](https://github.com/linuxmint/cinnamon/assets/1707476/f274e377-c948-469f-9570-6dbc0c1405a6)

### Steps to reproduce

Select Mint-Y theme, Appearance Dark, and open a Qt app, such as VLC video player version 3.0.19. Open configuration settings and notice text is missing for most options.

May require installation of Fedora 38 Cinnamon Spin, not sure. This is the package for my current version of VLC:
```
Source       : vlc-3.0.19-0.3.fc38.1.src.rpm
Repository   : @System
From repo    : rpmfusion-free
```

### Expected behavior

Text should be visible in Qt apps while system preference is set to Dark mode.

### Additional information

I'm using nVidia's linux x64 driver, this was reproduced with driver versions 535.86 and 535.98.

Are there other popular dark mode themes I should install and test?

**Comment by leigh123linux** [comment]
It works fine here.

```
$ rpm -q vlc
vlc-3.0.19-0.3.fc38.1.x86_64
```


Do yo have the the qt gnome integration packages installed.

```
$ rpm -qa qgnomeplatform\*
qgnomeplatform-qt6-0.9.1-8.fc38.x86_64
qgnomeplatform-qt5-0.9.1-8.fc38.x86_64
```

![image](https://github.com/linuxmint/cinnamon/assets/1299302/0f1f6c78-ced6-4bdb-a80b-3502ae730174)

![image](https://github.com/linuxmint/cinnamon/assets/1299302/e9559c8d-1a5e-4e77-9e58-4486d9cb50a8)

![image](https://github.com/linuxmint/cinnamon/assets/1299302/3812d2c1-f42a-4009-8233-bd67b50cd7c8)

**Comment by jrefior** [comment]
> Do yo have the the qt gnome integration packages installed.

Yes I do:
```
Installed Packages
Name         : qgnomeplatform-qt5
Version      : 0.9.1
Release      : 8.fc38
Architecture : x86_64
Size         : 608 k
Source       : qgnomeplatform-0.9.1-8.fc38.src.rpm

Installed Packages
Name         : qgnomeplatform-qt6
Version      : 0.9.1
Release      : 8.fc38
Architecture : x86_64
Size         : 522 k
Source       : qgnomeplatform-0.9.1-8.fc38.src.rpm
```

Thanks for testing.

**Comment by Quantum-mutnauQ** [comment]
Hello jrefior,

I had the prblem with the back round color and I fixt it in the qt design settings but on the mint destr. I think yours letter color is black and background color is elso black, it make is rely hart to read.

**Comment by jrefior** [comment]
Thanks Quantum, I think you're right that the display issue is black text on black background. I'm not sure how it happened, because it wasn't a problem in Qt apps with Mint-Y Dark prior to my recent upgrade to Fedora 38. (I've been using Mint-Y Dark and Qt apps on Fedora for years and through several major version upgrades.)

I don't know how to change Qt design settings, but now that I learned the problem is connected to the Mint-Y theme (it took me a while to figure that out) I have switched to a different theme for now as a workaround, and that's fine.
